src/NLog/Targets/DatabaseTarget.cs

```diff
@@ -533,7 +539,18 @@ protected override void Write(LogEventInfo logEvent)
         {
             try
             {
-                WriteEventToDatabase(logEvent);
+                var connectionString = BuildConnectionString(logEvent);
+
+                //Always suppress transaction so that the caller does not rollback logging if they are rolling back their transaction.
+                using (TransactionScope transactionScope = new TransactionScope(TransactionScopeOption.Suppress))
```

Comment:
this is another change then the PR suggests?

Reply:
Nope just me moving code around to optimize the batching-logic.

Reply:
OK thanks!